### PR TITLE
Move (rather than link) the wgrib2 executable and delete build dir

### PR DESCRIPTION
**Pull request by rdenham**
Follow up on #576 to also remove unnecessary build files as part of the wgrib2 installation that will still take place on focal images.

The build of wgrib2 creates a standalone executable. Rather than linking this file to `/usr/local/bin/wgrib2` I suggest we copy it and then remove the entire `/opt/grib2` directory. The `wgrib2` executable doesn't require any files from the build directory, and this corresponds to the authors build instructions in their `INSTALLATION`  file:

```
Try executing wgrib2

      wgrib2/wgrib2 -config

Copy wgrib2 to SOMEPLACE

      cygwin/linux/unix: cp wgrib2/wgrib2 SOMEPLACE
```

(Sorry I didn't do this in time to make the previous pull request).

### CODE REVIEW

File: scripts/install_wgrib2.sh [hunk 1]
```diff
@@ -32,10 +32,12 @@ cd grib2
 
 ## really someone needs to learn proper packaging conventions, but whatever
 CC=gcc FC=gfortran make
-ln -s /opt/grib2/wgrib2/wgrib2 /usr/local/bin/wgrib2
+cp /opt/grib2/wgrib2/wgrib2 /usr/local/bin/wgrib2
 
 # Clean up
 rm -rf /var/lib/apt/lists/*
+rm -rf /opt/grib2 
+
```
Comment (eitsupi) [inline scripts/install_wgrib2.sh:40]:
```suggestion
```

File: scripts/install_wgrib2.sh [hunk 2]
```diff
@@ -32,10 +32,12 @@ cd grib2
 
 ## really someone needs to learn proper packaging conventions, but whatever
 CC=gcc FC=gfortran make
-ln -s /opt/grib2/wgrib2/wgrib2 /usr/local/bin/wgrib2
+cp /opt/grib2/wgrib2/wgrib2 /usr/local/bin/wgrib2
```
Comment (eitsupi) [inline scripts/install_wgrib2.sh:35]: Is there a reason why it is cp and not mv?

**Review by eitsupi — COMMENTED** [review]
Thank you for working on this!
I know almost nothing about this software, so it really helps that you have looked into this.

File: scripts/install_wgrib2.sh [hunk 3]
```diff
@@ -32,10 +32,11 @@ cd grib2
...
 
 # Clean up
 rm -rf /var/lib/apt/lists/*
+rm -rf /opt/grib2
```
Comment (eitsupi) [inline scripts/install_wgrib2.sh:39]:
```suggestion
rm -rf /opt/grib2
```

File: scripts/install_wgrib2.sh (hunk 2 above)
Comment (rdenham) [reply scripts/install_wgrib2.sh:35]: No reason really.  I guess I sometimes do cp and then check before removing the source. But in this case mv would be just as good.

File: scripts/install_wgrib2.sh [hunk 4]
```diff
@@ -32,10 +32,11 @@ cd grib2
 
 ## really someone needs to learn proper packaging conventions, but whatever
 CC=gcc FC=gfortran make
-ln -s /opt/grib2/wgrib2/wgrib2 /usr/local/bin/wgrib2
+cp /opt/grib2/wgrib2/wgrib2 /usr/local/bin/wgrib2
```
Comment (eitsupi) [inline scripts/install_wgrib2.sh:35]:
```suggestion
mv /opt/grib2/wgrib2/wgrib2 /usr/local/bin/wgrib2
```

**Comment by eitsupi** [comment]
Now let's merge.
Thanks!

**Pull request merged**